.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
   docker-rust:
     docker:
       # Note: Let CI use our MSRV, rather than latest
-      - image: cimg/rust:1.78.0
+      - image: cimg/rust:1.81.0
```

Comment:
Does this change bump MSRV? If so, we also need to bump it in `common/Cargo.toml`.

Reply:
Thanks for pointing out, addressed in cdfb19760f91